ui/README.md

```diff
@@ -1,6 +1,6 @@
 # NumaFlow UI
 
-![Numaflow Image](../docs/images/Numa.svg)
+![Numaflow Image](../docs/architecture/assets/numa.svg)
```

Comment:
Not under `architecture` but directly under `docs`. The ones under `architecture` should also move over later.

Reply:
Done. should we move the others too?

Reply:
let's not move the architecture yet. We can keep the "Improvement Proposals" and the related architecture separate.